server/app/api/v1/charity/model.js

```diff
@@ -0,0 +1,35 @@
+import mongoose from 'mongoose';
+import validate from 'mongoose-validator';
+import Promise from 'bluebird';
+
+// Remove warning log from Bluebird due to Mongoose promises
+Promise.config({
+  warnings: false
+});
+
+// Set mongoose promises to use Bluebird promise library
+mongoose.Promise = Promise;
+
+const validURL = validate({
+  validator: 'isURL',
+  httpStatus: 400
+});
```

Comment:
This is a validation used to validate if `Charity.url` is a valid URL. It's passed to key `validate` that Mongoose accepts for validations. The object passed to `validate` is just following the requirements of `mongoose-validator` (that is a wrapper for `validator.js`). If the URL validation fails, and error is sent with a 400 status. 

Eventually I want to cut this dependency out and write our own validations.